index.js

```diff
@@ -11,53 +12,57 @@ app.get('/teams', (request, response) => {
 
 })
 
-app.get('/teams/', (request, response) => {
+
+//why do i not need this, is it cause above
+/* app.get('/teams/', (request, response) => {
 
     const allTeams = teams.map((team) => {
 
         return team
 
     })
-})
+}) */
```

Comment:
Turns out you don't need it. Always delete commented out code before committing, it's helpful during dev but clutters the repo otherwise.